catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/WhenBackgroundChangesToBrickTest.java

```diff
@@ -96,11 +102,25 @@ public void testCreateNewBackground() {
 		onView(withId(R.id.pocketpaint_drawing_surface_view)).perform(click());
 		pressBack();
 
+		waitForResult(1000, 10);
+
 		List<LookData> lookDataList =
 				ProjectManager.getInstance().getCurrentProject().getDefaultScene().getBackgroundSprite().getLookList();
 
 		assertEquals(1, lookDataList.size());
 
 		Intents.release();
 	}
+
+	private void waitForResult(int waitTimeMillis, int checkInterval) throws InterruptedException {
```

Comment:
waitForResult could be useful for other tests also, maybe add the method to testUtils instead and call it from there?